### PR TITLE
Updated chap 3 docs to show verbs and filters

### PR DESCRIPTION
**Pull request by aappddeevv**

### CODE REVIEW

**Comment by aappddeevv** [comment]
It looks like there are some dependency errors in the CI build that are unrelated to my changes. 

It's not clear this will complete all checks Ok.

File: docs/03/b.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+You need to be careful about handling state in your filter as the filter
+is used asynchronously.
+
+In your scala code, you need to add the filter to the async-http-client. You can do
+this by configuring a new Http executor.
+
+```scala
+val myhttp = Http.configure(_.addRequestFilter(EchoFilter()))
+...
+val responseFuture = myhttp(someRequest OK as.)
```
Comment (farmdawgnation) [inline docs/03/b.markdown]: This code looks incorrect. `as.` doesn't do anything does it?

File: docs/03/b.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    // This just illustrates deriving a new request object
+    // from the exsiting. The request object here is the async-http-client
+    // request object.
+    val builder = new RequestBuilder(ctx.getRequest)
+    // Potentially modify the original request...
+    val newRequest = builder.build
+    val requestNum = counter.incrementAndGet
+
+    // Print the request to the console.
+    println(s"Request \${requestNum} of length: \${newRequest.getContentLength}").
```
Comment (farmdawgnation) [inline docs/03/b.markdown]: I'm not clear on the reasoning behind the `\$`. What's the story there?

File: docs/03/b.markdown (hunk 1 above)
Comment (aappddeevv) [inline docs/03/b.markdown]: I'd drop this PR. I went ahead and create a place for some notes: https://aappddeevv.gitbooks.io/my-dispatch-user-notes/content/clients.html.

**Comment by farmdawgnation** [comment]
Dropping this PR at the author's request.

**Pull request closed without merging**